tests/sentry/dynamic_sampling/test_tasks.py

```diff
@@ -255,7 +255,12 @@ def test_prioritise_transactions_simple(self, get_blended_sample_rate):
         """
         get_blended_sample_rate.return_value = 0.25
 
-        with self.options({"dynamic-sampling.prioritise_transactions.load_rate": 1.0}):
+        with self.options(
+            {
+                "dynamic-sampling.prioritise_transactions.load_rate": 1.0,
+                "dynamic-sampling.prioritise_transactions.rebalance_intensity": 7.0,
```

Comment:
Why is the intensity `7.0`? Shouldn't it be in the range `[0.0, 1.0]`?

Reply:
hmm, good catch... it should have been 0.7... looking at the test to see what it does, since it obviously should not do anything with 7.0 .

Reply:
The test was checking transactions, so the option had no effect, removed.
Thanks for spotting it.